pkg/kn/commands/service/describe.go

```diff
@@ -321,7 +321,7 @@ func completeWithUntargetedRevisions(client clientservingv1.KnServingClient, ser
 			continue
 		}
 		revisionsSeen.Insert(revision.Name)
-		newDesc, err := newRevisionDesc(&revision, nil, service)
+		newDesc, err := newRevisionDesc(revision.DeepCopy(), nil, service)
```

Comment:
Couldn't we just past the revision by calue, not by reference ?  I think this would be even cleaner. wdyt ?

Reply:
That will modify the parameter type in `newRevisionDesc` function. It brings some changes which need to test.